### ISSUE TITLE
Watch for new confirmed transactions, related to specific account

### ISSUE THREAD

**Issue by takahser** [nimiq-network/core#444]
## New issue checklist

- [ x ] I have read all of the [`README`](https://github.com/nimiq-network/core/blob/master/README.md) 
- [ x ] I have searched [existing issues](https://github.com/nimiq-network/core/issues?q=is%3Aissue+sort%3Acreated-desc) and **this is not a duplicate**.

## Question or Feature Request

How can I watch for new transactions (incoming & outgoing), related to a specific account, in a pub/sub kind of way?

- I identified `_onTransactionAdded`, but this just notifies me whenever new txs are added to the mempool, rather than being mined
- also, I discovered Mempool.getTransactions however, this is returning just a promise, so I won't get notified for future transactions
- the only option I currently see is to listen to the `head-changed` event on the blockchain and for each block, manually checking whether it contains any transactions related to the specific account I'm watching.

Please let me know, if there is an easier way now or in the future.

Thanks.

**Comment by sisou** [comment]
The nano consensus has methods for subscribing, here: https://github.com/nimiq-network/core/blob/master/src/main/generic/consensus/nano/NanoConsensus.js#L32.

Once subscribed, the nano mempool fires `transaction-mined` events, from here: https://github.com/nimiq-network/core/blob/master/src/main/generic/consensus/nano/NanoMempool.js#L139. 

Example would be:
```javascript
cost consensus = await Nimiq.Consensus.nano();
consensus.addSubscriptions(new Nimiq.Address.fromUserFriendlyAddress('NQxx xxxx...');
consensus.mempool.on('transaction-mined', (tx) => console.log(tx));
```

**Comment by takahser** [comment]
Neat!
Is there any way to get the height of the block, the tx was baked into?

**Comment by sisou** [comment]
Yes, the `blockheader`, the second parameter to the callback of the `transaction-mined` event, is a `Nimiq.BlockHeader` and has the `height` property.
